templates/endpoints_v2/llm-serve-aws.yaml

```diff
@@ -1,13 +0,0 @@
-applications:
```

Comment:
2 comments - 
1. Missing changes on service deploy instructions
2. Can we still keep one file here for custom model deployment?